core/iwasm/aot/aot_runtime.h

```diff
@@ -293,6 +293,7 @@ typedef struct AOTMemoryInstance {
     uint32 num_bytes_per_page;
     uint32 cur_page_count;
     uint32 max_page_count;
+    /*TODO:memory_data_size should be uint64*/
     uint32 memory_data_size;
```

Comment:
not sure if we should change `uint32 memory_data_size` -> `uint64 memory_data_size`?